backend/src/ee/routes/v1/ssh-host-router.ts

```diff
@@ -96,10 +97,20 @@ export const registerSshHostRouter = async (server: FastifyZodProvider) => {
         hostname: z
           .string()
           .min(1)
+          .trim()
           .refine((v) => isValidHostname(v), {
             message: "Hostname must be a valid hostname"
           })
           .describe(SSH_HOSTS.CREATE.hostname),
+        alias: z
+          .string()
+          .trim()
+          .nullable()
+          .default(null)
+          .refine((v) => v == null || slugify(v) === v, {
+            message: "Alias must be a valid slug"
+          })
```

Comment:
let's use `slugSchema` here to standardize slug format
<img width="857" alt="image" src="https://github.com/user-attachments/assets/e541d1fd-5497-4dfb-8198-52bd93c5ac65" />

Reply:
Ah didn't realize there was already `slugSchema`; sounds good.